src/regexmanager.cpp

```diff
@@ -177,26 +162,27 @@ void RegexManager::quote_and_highlight(std::string& str,
 
 	for (unsigned int i = 0; i < regexes.size(); ++i) {
 		const auto& regex = regexes[i].first;
-		if (regex == nullptr) {
```

Comment:
Now that first element of the pair is a pointer again, I think this edit can be reverted. `x != nullptr` is way more explicit than `!x`.